changelogs/fragments/4953-listen-ports-facts-extend-output.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - listen_ports_acts - add ``-a`` option to ``netstat`` and ``ss``. This shows both listening and non-listening (for TCP this means established connections) sockets (https://github.com/ansible-collections/community.general/issues/4762, https://github.com/ansible-collections/community.general/pull/4953). Introduce new param ``include_non_listening`` to turn this behavior on or off (default off). Also return ``state`` and ``foreign_address`` when ``include_non_listening`` is set.
```

Comment:
```suggestion
  - listen_ports_acts - add new ``include_non_listening`` option which adds ``-a`` option to ``netstat`` and ``ss``. This shows both listening and non-listening (for TCP this means established connections) sockets, and returns ``state`` and ``foreign_address`` (https://github.com/ansible-collections/community.general/issues/4762, https://github.com/ansible-collections/community.general/pull/4953).
```

Reply:
added and fixed typo in module name: listen_ports_**f**acts